lib/chef/provisioning/vsphere_driver/vsphere_helpers.rb

```diff
@@ -8,6 +9,12 @@ class VsphereHelper
     end
 
     def initialize(connect_options, datacenter_name)
+      if ENV['https_proxy']
```

Comment:
I'd do the same with `HTTP_PROXY` and make `https_proxy` uppercase which should not matter on windows but will on linux.